addons/docs/src/frameworks/angular/compodoc.ts

```diff
@@ -71,6 +71,15 @@ const mapItemToSection = (key: string, item: Method | Property): string => {
   }
 };
 
+export const findComponentByName = (name?: string, compodocJson?: CompodocJson) => {
+  return (
+    name &&
+    compodocJson &&
+    (compodocJson.components.find((c: Component) => c.name === name) ||
+      compodocJson.directives.find((c: Directive) => c.name === name))
+  );
```

Comment:
If `name` and `compodocJson` are both necessary for a proper return value.. can't you change the arguments to 
```
(name: string, compodocJson: CompodocJson) 
```

This would result in 

```typescript
export const findComponentByName = (name: string, compodocJson: CompodocJson) => {
  return (
      compodocJson.components.find((c: Component) => c.name === name) ||
      compodocJson.directives.find((c: Directive) => c.name === name)
  );
}
```
which is kinda easier to read?

Reply:
Good call. there's no 😂 response on github ❤️